uber/reports.py

```diff
@@ -32,11 +32,18 @@ def run(self, out, session, *filters, order_by=None, badge_type_override=None):
 
 class PrintedBadgeReport(ReportBase):
     """Generate a CSV file of badges which do not have customized information"""
-    def __init__(self, badge_type, include_badge_nums=True):
+    def __init__(self, badge_type, include_badge_nums=True, range=None):
         self._badge_type = badge_type
         self._include_badge_nums = include_badge_nums
+        self._range = range
 
     def run(self, out, session):
         badge_range = c.BADGE_RANGES[self._badge_type]
-        for badge_num in range(badge_range[0], badge_range[1] + 1):
-            self.write_row([badge_num], out)
+        min_badge_num = max([badge_range[0]] + ([self._range[0]] if self._range else []))
```

Comment:
if the range override self._range is NOne, then yes, they are which means they just return that one argument.